### PR TITLE
Feature/gs/onrmp 1278 wire l1 data

### PR DESCRIPTION
**Pull request by gsteenkamp89**

### CODE REVIEW

File: package/src/steps/IframeView/BodyIframeView.tsx [hunk 1]
```diff
@@ -87,7 +92,41 @@ const BodyIframeView: React.FC<BodyIframeViewType> = (props) => {
...
+            type: "paymentProgress",
+            progress: 80,
+            tokenIn: {
+              name: "Wrapped Ether",
+              address: "0xc778417E063141139Fce010982780140Aa0cD5Ab",
+              symbol: "WETH",
+              decimals: 18,
+              chainId: 4,
+              logoURI:
+                "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2/logo.png",
```
Comment (leeroyhendriks) [inline package/src/steps/IframeView/BodyIframeView.tsx:109]: Temporary?

File: package/src/steps/IframeView/index.tsx [hunk 2]
```diff
@@ -69,6 +71,42 @@ const IframeView: React.FC<{
...
+                  type: "paymentProgress",
+                  progress: 80,
+                  tokenIn: {
+                    name: "Wrapped Ether",
+                    address: "0xc778417E063141139Fce010982780140Aa0cD5Ab",
+                    symbol: "WETH",
+                    decimals: 18,
+                    chainId: 4,
+                    logoURI:
+                      "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2/logo.png",
```
Comment (leeroyhendriks) [inline package/src/steps/IframeView/index.tsx:99]: Idem.

**Review by leeroyhendriks — APPROVED** [review]

File: package/src/steps/IframeView/BodyIframeView.tsx (hunk 1 above)
Comment (gsteenkamp89) [reply package/src/steps/IframeView/BodyIframeView.tsx:109]: This PR is supposed to be a draft 😅 My bad

File: package/src/TransactionContext/hooks/index.tsx [hunk 3]
```diff
@@ -1,4 +1,3 @@
 export * from "./useTransactionContext";
 export * from "./useTransactionCtxWallets";
 export * from "./useTransactionCtxActions";
-export * from "./useTranasactionCtxInit";
```
Comment (gaelicgod) [inline package/src/TransactionContext/hooks/index.tsx:4]: add a newline after this please.

File: package/src/TransactionContext/initialState.tsx [hunk 4]
```diff
@@ -1,46 +1,54 @@
...
-    quoteId: "0",
+    quoteId: "",
   },
   tokenIn: {
     name: "",
     address: "",
     symbol: "",
     decimals: 18,
-    chainId: 4,
+    chainId: 3,
```
Comment (gaelicgod) [inline package/src/TransactionContext/initialState.tsx:31]: Would this have to be manually update before going live?

Comment (gsteenkamp89) [reply package/src/TransactionContext/initialState.tsx:31]: I think a safer default value would be testnet. And so only if we actually pull in a mainnet token from the BE response do we overwrite the test-env-first value.

**Pull request merged**